armada/api/__init__.py

```diff
@@ -11,3 +11,156 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from armada.conf import default
+default.register_opts()
```

Comment:
Consider moving this into your API service initialization logic instead of running each time modules are imported.